client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -230,6 +231,7 @@ class PurchaseMeta extends Component {
 		if (
 			! canEditPaymentDetails( purchase ) ||
 			! isPaidWithCreditCard( purchase ) ||
+			! cardProcessorSupportsUpdates( purchase ) ||
```

Comment:
Should this check also be included when rendering the [Edit Payment Button](https://github.com/Automattic/wp-calypso/blob/f5d600101de23b6e22bed833cc9237b969811046/client/me/purchases/manage-purchase/index.jsx#L199) as well?
<img width="772" alt="screen shot 2018-01-02 at 2 43 13 pm" src="https://user-images.githubusercontent.com/6458278/34474075-394f769a-efce-11e7-8df2-4a20ff2f41c9.png">